File: src/Match.java
import java.util.ArrayList;
import java.util.Date;
import java.util.List;


public class Match {
    private Date date;

    private String heure;

    private int[] score = new int[2];

    private List<Equipe> equipe = new ArrayList<Equipe> ();

    public String toString() {
        return heure;
    }

    public void setScore(List<Integer> scores) {
    }

    public Match(Date date, String heure) {
    }

    public Equipe gagnant() {
        return null;
    }

}

File: modele/Match.php
<?php
class EMatch
{
    private $date;
    private $heure;
    private $scoreEquipe = array();
    function __construct($date, $heure, $equipe1, $equipe2){
        $this->date = $date;
        $this->heure = $heure;
        $this->scoreEquipe[$equipe1] = 0;
        $this->scoreEquipe[$equipe2] = 0;
    }
    public function toString()
    {
        return $this->heure;
    }
    public function setScore($scores)
    {
    }
    public function gagnant()
    {
    }
}

?>